src/args/argument_list.go

```diff
@@ -15,6 +17,7 @@ type ArgumentList struct {
 	Hostname               string `default:"localhost" help:"The agent node Hostname or IP address to connect to"`
 	Port                   string `default:"8500" help:"Port to connect to agent node"`
 	Token                  string `default:"" help:"ACL Token if token authentication is enabled"`
+	Timeout                int    `default:"0" help:"Timeout for an API call"`
```

Comment:
would make sense to use the golang time syntax here as is used in `interval` or `timeout` ? i.e: `30s`

Reply:
I have had to add it to all the unit tests, because the zero value of string is "" and the default we set is only honored when integration.new is called.